### DISCUSSION TITLE
Nextflow grammer rules files

### DISCUSSION THREAD

**Discussion by obonyojimmy** [nextflow-io/nextflow#3182] [Q&A]
Am trying to find the "Grammer" specification nextflow is using , is there a source file (link) i can get this from ?

**Comment by mribeirodantas** [comment]
The closest to a grammar file for the Nextflow domain-specific language (DSL) that we have is the official documentation on DSL2. You can check that by clicking [here](https://www.nextflow.io/docs/latest/dsl2.html).
